extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/VertxHttpRecorder.java

```diff
@@ -233,6 +233,19 @@ public void handle(AsyncResult<Void> event) {
                 "Listening on: http://%s:%s", httpServerOptions.getHost(), httpServerOptions.getPort()));
     }
 
+    private static int calculateDefaultIOThreads() {
+        //we only allow one event loop per 4mb of ram at the most
+        //its hard to say what this number should be, but it is also obvious
+        //that for constrained environments we don't want a lot of event loops
+        //lets start with 4mb and adjust as needed
+        int recommended = Runtime.getRuntime().availableProcessors() * 2;
```

Comment:
This should probably use `org.wildfly.common.cpu.ProcessorInfo#availableProcessors` at least for now.

Reply:
@stuartwdouglas do you plan to address this comment?